test/check-machines-create

```diff
@@ -1088,12 +1088,13 @@ class TestMachinesCreate(VirtualMachinesCase):
             # because we might end up checking the text for the new state the momment it disappears
             def waitStateWentAway(selector, state):
                 try:
-                    b.wait_text_not(selector, state)
+                    with b.wait_timeout(10):
+                        b.wait_text_not(selector, state)
                     return True
                 except Error:
                     return False
 
-            wait(lambda: waitStateWentAway(selector, before), delay=3)
+            wait(lambda: waitStateWentAway(selector, before), tries=10, delay=3)
```

Comment:
We could also do a plain:
wait(lambda: b.text(selector) != state, tries=10, delay=3) and remove waitStateWentAway

Reply:
You are right, this is even better. Admittedly I was not sure what exactly this code is supposed to achieve. I'll wait for this round of test results and push that, it's much easier to read/understand.

Reply:
Previous testing round looks promising. I applied this simplification.